js/src/attribute-mapping/index.scss

```diff
@@ -3,4 +3,48 @@
 	&__table-footer {
 		padding: $grid-unit-20;
 	}
+
+	&__radio-control {
+		margin-top: $grid-unit-20;
+		.help-popover {
+			// hack for positioning correctly the popover in the modal radio control
+			.components-popover {
+				position: absolute;
+			}
+		}
+	}
+
+	&__rule-modal {
+
+		.app-select-control {
+			margin-bottom: $grid-unit;
+			select.components-select-control__input {
```

Comment:
Part of the `<select>` is no reaction when clicking. I guess it would be nicer to make it clickable for the whole rendering area.

![Kapture 2022-09-23 at 17 24 58](https://user-images.githubusercontent.com/17420811/191939402-f848b998-9220-49c2-91dc-cf461d8db9f1.gif)